test/deploy/rackhd_source_install.py

```diff
@@ -281,14 +213,11 @@ def test05_install_rackhd_config_files(self):
         fit_common.remote_shell('mkdir -p ~/src/on-http/static/swagger-ui')
 
     def test06_startup(self):
-        print "Start services."
-        startup = open('startup.sh', 'w')
-        startup.write('cd ~/;nf start&\n')
-        startup.close()
-        fit_common.scp_file_to_ora('startup.sh')
+        print "**** Start services."
         self.assertEqual(fit_common.remote_shell("chmod 777 startup.sh;/etc/init.d/isc-dhcp-server restart")['exitcode'], 0, "dhcp startup failure.")
```

Comment:
I should change that. It won't fail because the last command in the line generates the exit code.